modules/manage/partials/remote-read-replicas.adoc

```diff
@@ -36,11 +34,12 @@ endif::[]
 
 You need the following:
 
```

Comment:
In the Important note above, you could change the releases now from 23.x to 24.x.